climakitae/visualize.py

```diff
@@ -0,0 +1,136 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import matplotlib.pyplot as plt
+from matplotlib import cm
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import geoviews as gv
+
+import holoviews as hv
+from holoviews import opts
+import hvplot.pandas
+import hvplot.xarray
+
+#####################################################################
+
+
```

Comment:
flagging here that in the initial notebook some datapoints failed to visualize

Reply:
probably due to a ValueError where if some checks through the lmoments3 package didn't go through, they come back as a nan value for that particular grid cell (rather than causing none of the visualization to run). another good thing to explain in documentation, why a user might be seeing that!